### ISSUE TITLE
App fails with WebSocket rejected HTTP 404

### ISSUE THREAD

**Issue by fatestapestry** [jacoduplessis/luno_streams#4]
Broken by latest fix:

```
(luno_streams) me@mypc-laptop:~/develop/luno_streams$ luno_streams -l DEBUG -d 15 --app my_key my_secret XBTZAR
Open the app in your browser: http://localhost:8011
Running websocket server on port 8010
INFO:luno_streams:[XBTZAR] Connecting to wss://ws.luno.com/api/1/stream/xbtzar...
Traceback (most recent call last):
  File "/home/me/.virtualenvs/luno_streams/bin/luno_streams", line 11, in <module>
    load_entry_point('luno-streams', 'console_scripts', 'luno_streams')()
  File "/home/me/develop/luno_streams/luno_streams/cli.py", line 70, in main
    loop.run_until_complete(asyncio.gather(*tasks))
  File "/usr/lib/python3.7/asyncio/base_events.py", line 584, in run_until_complete
    return future.result()
  File "/home/me/develop/luno_streams/luno_streams/cli.py", line 54, in run_updater
    await updater.run()
  File "/home/me/develop/luno_streams/luno_streams/updater.py", line 65, in run
    await self.connect()
  File "/home/me/develop/luno_streams/luno_streams/updater.py", line 50, in connect
    self.websocket = await websockets.connect(self.url)
  File "/home/me/.virtualenvs/luno_streams/lib/python3.7/site-packages/websockets/client.py", line 547, in __await_impl__
    extra_headers=protocol.extra_headers,
  File "/home/me/.virtualenvs/luno_streams/lib/python3.7/site-packages/websockets/client.py", line 296, in handshake
    raise InvalidStatusCode(status_code)
websockets.exceptions.InvalidStatusCode: server rejected WebSocket connection: HTTP 404
```

was working before the latest update

```
(luno_streams) me@mypc:~/develop/luno_streams$ pip freeze
-e git+git@github.com:jacoduplessis/luno_streams.git@da256788558480765f7afb0d9d64ebd8716a00ad#egg=luno_streams
websockets==8.1
```

**Comment by jacoduplessis** [comment]
Seems Luno requires uppercase in their URLs. Fixed in 0.1.4 that was just uploaded to PYPI.

**Comment by fatestapestry** [comment]
Yeah, it's working - thank you, sir!  My apologies that the bug didn't come with a PR.  Busy weekend.
